README.md

```
@@ -77,24 +83,25 @@ x402 test https://api.example.com/resource --keystore ~/.foundry/keystores/my-wa
 
 ## Sample Output
 
-### Health Check
+### Health Check (EVM + Solana)
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)